### PR TITLE
[14.0] Fix account_invoice_export resend button visibility

### PR DESCRIPTION
**Pull request by TDu**
The resend button was not displayed when needed.
Looks like some parts of a commit on 13 missing.

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by ajaniszewska-dev** [comment]
@TDu  code LG but test fails..

**Comment by sebalix** [comment]
I don't know why `pip` tries to compile `PyMuPDF` from the tarball as there is a wheel package available... a new release of this package has been published on February 3, could be related.

**Review by i-vyshnevska — APPROVED** [review]

File: account_invoice_export/views/account_move.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
                     name="export_invoice"
                     string="Resend eBill"
                     type="object"
-                    attrs="{'invisible':['|', '|', '|', ('send_through_http', '=', False), ('state', '!=', 'posted'), '|', ('invoice_export_confirmed', '=', True), ('invoice_exported', '=', False), ('move_type', 'not in', ('out_invoice', 'out_refund'))]}"
```
Comment (simahawk) [inline account_invoice_export/views/account_move.xml:26]: tip: when domains start to be messy as this, use a computed flag :wink:

**Comment by simahawk** [comment]
> PyMuPDF

https://github.com/OCA/edi/pull/522#issuecomment-1033462193

**Comment by gdgellatly** [comment]
@sebalix from personal experience the new release is problematic. Probably want to pin at 1.19.4, works for me anyway.

\cc @TDu

**Comment by alexis-via** [comment]
My PR https://github.com/OCA/edi/pull/522 will make the v14 branch green again. So we should merge it in priority.

**Pull request merged**